docs/ref/contrib/postgres/indexes.txt

```diff
@@ -33,4 +33,18 @@ available from the ``django.contrib.postgres.indexes`` module.
     <https://www.postgresql.org/docs/current/static/btree-gin.html>`_ on
     PostgreSQL. You can install it using the
     :class:`~django.contrib.postgres.operations.BtreeGinExtension` migration
-    operation.
+    operation. The parameter ``fastupdate`` takes a boolean value (default in
+    PostgreSQL is True corresponding to 'on'), and ``gin_pending_list_limit``
```

Comment:
I'm omit details about gin_pending_list_limit and link to https://www.postgresql.org/docs/current/static/runtime-config-client.html#GUC-GIN-PENDING-LIST-LIMIT.